action-variables.tf

```diff
@@ -1,3 +0,0 @@
-variable "git" {}
```

Comment:
We going to inject these manually into our unit tests from now on? Should make it cleaner for sure.

Reply:
I really don't know, I was trying it out, but I haven't gone and converted everything to use the newer version.

I was trying it out for better compatibility with goland which is what I think we need. Ultimately it's not valid terraform code if all of the referenced variables don't exist in the repo and it was just invisible to me because I didn't have a proper terraform integration in my IDE.